guacamole/src/main/java/org/apache/guacamole/tunnel/InputStreamInterceptingFilter.java

```diff
@@ -112,9 +112,8 @@ private void readNextBlob(InterceptedStream<InputStream> stream) {
             // End stream if no more data
             if (length == -1) {
 
-                // Close stream, send end if the stream is still valid
-                if (closeInterceptedStream(stream))
-                    sendEnd(stream.getIndex());
+                // Close stream
+                closeInterceptedStream(stream);
```

Comment:
Why did you get rid of the "end" instruction here?

Reply:
The end instruction closes the writer, so without deleting that line, each subsequent chunk would just rewrite the previous. The stream is now closed when the upload is complete by ManagedFileUpload's stream state.

Reply:
Ah gotcha. One of the reasons that I brought this up is that the block comment for this function still specifies that the an "end" instruction will automatically be sent if the end of the InputStream is reached, which is no longer correct. The comment should be updated accordingly.

Reply:
Thanks! Just committed the changes you requested.

Reply:
@elijahnicpon I'm not seeing where the `end` instruction is sent now that this `sendEnd()` call has been removed. I do see that `ManagedFileUpload` updates its own understanding of the stream state, but the associated stream will still be active and consuming resources within guacd until `end` is received.

Can you point to where the relevant `end` instruction is sent from, if it will not be here?

Reply:
You're right. I could've sworn I traced that up but I definitely made a mistake there. Thanks!